tests/Fixer/Import/NoUnusedImportsFixerTest.php

```diff
@@ -29,6 +29,9 @@ final class NoUnusedImportsFixerTest extends AbstractFixerTestCase
      */
     public function testFix($expected, $input = null)
     {
+        $this->fixer->configure([
+            'treat_same_namespace_as_unused' => false,
```

Comment:
Why all tests should use non-default configuration?

Please leave all original tests as they were and add new ones with this option set to `false`.

Reply:
done @kubawerlos

Reply:
ping @kubawerlos :)